COLLECTOR_PARAMS.md

```diff
@@ -8,8 +8,8 @@
 - contains the b64 url encoded string of the JSON that was sent via `liveConnect.push` function
 ### `duid`
 - contains the LiveConnect managed first party identifier, in the `${apexDomainHash}--${ULID}`
-### `tna`
```

Comment:
I'm fine with the change. But given that we need to maintain backwards compat, I'm not sure this will be any less confusing